src/server.ts

```diff
@@ -1,24 +1,22 @@
 import { APP_BASE_HREF } from '@angular/common';
 import { CommonEngine } from '@angular/ssr/node';
-// @ts-ignore
 import { render } from '@netlify/angular-runtime/common-engine';
 import express from 'express';
-import { fileURLToPath } from 'node:url';
 import { dirname, join, resolve } from 'node:path';
-import bootstrap from './src/main.server';
-
+import { fileURLToPath } from 'node:url';
+import bootstrap from './main.server';
 
-const commonEngine = new CommonEngine()
+const commonEngine = new CommonEngine();
```

Comment:
Well, SSR seems to be useless here (there's only 1 route that is already being SSG), it was part of my question on the main message `If you need SSR, I'll migrate to the new server API` :p

I'll probably do it anyway, it will force to use the new `ServerRoute` to setup the SSG as `prerender` is not being used with `outputMode` from what I understood